azure-pipelines.yml

```diff
@@ -53,9 +49,14 @@ steps:
                   echo "opam \$@" > opam2_local &&
                   chmod a+x opam2_local ;;
         *) exit 1 ;;
-      esac
+      esac;;
+      opam install ocamlformat
```

Comment:
Make this a separate step; and you need to use the wrapper script `opam2_local` rather than `opam`.